<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/iconfinder_50_Cloud_Arrow_Up_183463.png</normaloff>icon/iconfinder_50_Cloud_Arrow_Up_183463.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>9</y>
      <width>461</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_S"/>
    <addaction name="action_B"/>
    <addaction name="action_Y"/>
    <addaction name="actionpdf"/>
    <addaction name="action_Pdf"/>
    <addaction name="action_O"/>
    <addaction name="action_C"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_Z"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>云服务(&amp;Y)</string>
    </property>
    <addaction name="action_L"/>
    <addaction name="action_D"/>
    <addaction name="action_Q"/>
    <addaction name="action_T"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_N"/>
   <addaction name="action_S"/>
   <addaction name="action_B"/>
   <addaction name="action_Y"/>
   <addaction name="actionpdf"/>
   <addaction name="action_Pdf"/>
   <addaction name="action_O"/>
   <addaction name="separator"/>
   <addaction name="action_Z"/>
   <addaction name="action"/>
   <addaction name="separator"/>
   <addaction name="action_L"/>
   <addaction name="action_Q"/>
   <addaction name="action_T"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_N">
   <property name="icon">
    <iconset>
     <normaloff>../icon/document-new.svg</normaloff>../icon/document-new.svg</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset>
     <normaloff>../icon/document-save.svg</normaloff>../icon/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_B">
   <property name="icon">
    <iconset>
     <normaloff>../icon/edit-copy.svg</normaloff>.../icon/edit-copy.svg</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;B)</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_O">
   <property name="icon">
    <iconset>
     <normaloff>../icon/document-open.svg</normaloff>../icon/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_C">
   <property name="icon">
    <iconset>
     <normaloff>../icon/application-exit.svg</normaloff>../icon/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_X">
   <property name="icon">
    <iconset>
     <normaloff>../icon/system-shutdown.svg</normaloff>../icon/system-shutdown.svg</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Z">
   <property name="icon">
    <iconset>
     <normaloff>../icon/go-previous.svg</normaloff>../icon/go-previous.svg</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>../icon/edit-find.svg</normaloff>../icon/edit-find.svg</iconset>
   </property>
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionpdf">
   <property name="icon">
    <iconset>
     <normaloff>../icon/html.svg</normaloff>../icon/html.svg</iconset>
   </property>
   <property name="text">
    <string>导出Html(&amp;H)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_Pdf">
   <property name="icon">
    <iconset>
     <normaloff>../icon/document-print.svg</normaloff>../icon/document-print.svg</iconset>
   </property>
   <property name="text">
    <string>导出Pdf(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_L">
   <property name="icon">
    <iconset>
     <normaloff>../icon/folder-cloud.svg</normaloff>../icon/folder-cloud.svg</iconset>
   </property>
   <property name="text">
    <string>登录或注册(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_D">
   <property name="icon">
    <iconset>
     <normaloff>../icon/dialog-cancel.svg</normaloff>../icon/dialog-cancel.svg</iconset>
   </property>
   <property name="text">
    <string>注销(&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_T">
   <property name="icon">
    <iconset>
     <normaloff>../icon/view-refresh.svg</normaloff>../icon/view-refresh.svg</iconset>
   </property>
   <property name="text">
    <string>同步(&amp;T)</string>
   </property>
  </action>
  <action name="action_Q">
   <property name="icon">
    <iconset>
     <normaloff>../icon/edit-download.svg</normaloff>../icon/edit-download.svg</iconset>
   </property>
   <property name="text">
    <string>选择云端文件(&amp;Q)</string>
   </property>
  </action>
  <action name="action_Y">
   <property name="icon">
    <iconset>
     <normaloff>../icon/text-x-generic.svg</normaloff>../icon/text-x-generic.svg</iconset>
   </property>
   <property name="text">
    <string>预览(&amp;Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
